### DISCUSSION TITLE
I want to leave SD overnight to generate 50 images with Prompt 1 and 50 images with Prompt 2.  Can this addon do this?

### DISCUSSION THREAD

**Discussion by xjdeng** [adieyal/sd-dynamic-prompts#731] [Q&A]
Not sure what's the terminology for what I'm trying to do here and what settings I'll need to tweak to accomplish this but I'll describe using examples.

I've been playing around with this addon for over a year in the more typical way: inserting a wildcard somewhere in my prompt and it works great.  But I'm wondering if this addon can be leveraged for a different but related purpose.

As you know, A1111 can generate up to 100 images at a time which is quite useful when leaving it running overnight (more if you increase the batch size if your GPU allows it.)  Let's say I have 2 different prompts that I want to generate images for and I want to split it 50/50.  Basically, I want A1111 to generate 50 images using Prompt 1 and then 50 more images using Prompt 2.

Is there a way to do this with this addon?  If I make a text file (call it wildcards.txt) and put Prompt 1 on a line then Prompt 2 on another line then put \_\_wildcards\_\_ in my prompt and crank the batch count to 100, it'll pick either Prompt 1 or Prompt 2 at random each time and not only is there no guarantee I'll get 50 of each, they'll all be jumbled together rather than having the first 50 files using Prompt 1 and the next 50 using Prompt 2.  I want to wake up in the morning, open my outputs folder, and copy the first 50 files (using Prompt 1) to a collection then copy the next 50 files (using Prompt 2) to another collection rather than having to inspect and select each one individually, not to mention it may not be a 50:50 ratio.

Have I explained what I'd like to do clearly?  If so, is there any setting I'm missing here that allows me to do exactly what I'm trying to accomplish?

**Comment by Footy0604** [comment]
I don't know an answer but I would like to have a way for a batch of images to do: 10 first option in wild card, 10 second option in wild card, 10 third option in wild card. Ex.

A {red|blue|yellow} car.

Maybe some numeric + flag such as:

A {10%%red|blue|yellow} car. <--This would resule in 10 images of red car, follwed by 10 images of blue car and then 10 yellow car.
